packages/ddp-client/client/queueStubsHelpers.js

```diff
@@ -121,7 +121,7 @@ export const loadAsyncStubHelpers = () => {
         Meteor._setImmediate(() => {
           if (hasStub && !finished) {
             console.warn(
-              `Method stub (${name}) took too long and could cause unexpected problems. Learn more at https://github.com/zodern/fix-async-stubs/#limitations`
+              `Method stub (${name}) took too long and could cause unexpected problems. Learn more at https://v3-migration-docs.meteor.com/breaking-changes/call-x-callAsync.html#what-are-the-limitations-of-call-meteor-callasync`
```

Comment:
It's probably out of scope for this PR, but if we are going to link to the docs from console messages, we probably need a solution to make the links stable over time.

 For example, something like `docs.meteor.com/link/long-method-stub` that redirects to the current docs.

Reply:
That's a good idea